terraform/development/vpc/versions.tf

```diff
@@ -1,6 +1,6 @@
 terraform {
   # Must be above 1.9.0 to allow cross-object referencing for input variable validations
-  required_version = ">=1.9.0, < 2.0.0"
```

Comment:
Please undo this change so I can merge this PR without triggering the workflow. The workflow run failed because it cannot access secrets required.

Reply:
Reverted as requested.

Are you sure you're okay with merging changes to the workflow without testing it out first?

Reply:
Yes, let's give it a go - this isn't a production workflow. I'll raise another PR to test it out and fix forward.